analyses/cell-type-dsrct/references/tumor-marker-genes.tsv

```diff
@@ -0,0 +1,9 @@
+cell_type	gene_symbol	ensembl_gene_id	source
+tumor	ST6GALNAC5	ENSG00000117069	internal
+tumor	CACNA2D2	ENSG00000007402	internal
+tumor	IQCJ-SCHIP1	ENSG00000283154	internal
+tumor	PTPRQ	ENSG00000139304	Internal
```

Comment:
Two articles describe them but its buried in the supplementary. Other than that, we are seeing it clearly in our datasets. We have one publicly available DSRCT PDX single-cell data set that I can compare against control samples where we'd see these same genes too.

Reply:
It is probably worth including the references even if they are buried in the supplement. You can also include your lab  as an unreferenced source, if you think that would be more useful... I think we just want to avoid "internal" as a reference as that makes it harder to trace.